CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ would copy internal references to the generators and would lead to double free o
   These types should be moved instead of copied, and move constructors and operators are now defined
   for them. 
 - Improved MT19937 initialization and generation performance.
-- Removed references to and workarounds for deprecated hcc
+- Removed references to and workarounds for deprecated hcc.
```

Comment:
Please separate any changelog entries for this PR to ROCm 6.1, not ROCm 6.0.

Reply:
Fixed in 2fa3c1b